deploy/kubeturbo-operator/deploy/crds/charts_v1alpha1_kubeturbo_crd.yaml

```diff
@@ -101,9 +101,10 @@ spec:
               type: object
               description: Disable features
               properties:
-                disabledFeatures:
-                  description: Feature names
-                  type: string, quote
+                default:
+                  type: string
+                additionalProperties:
+                  type: boolean
```

Comment:
According to [this](https://swagger.io/docs/specification/data-models/dictionaries/), shouldn't the `featureGates` map be defined as:

```
           featureGates:
              type: object
              description: Disable features
              properties:
                default:
                  type: string
              additionalProperties:
                type: boolean
```

* The `additionalProperties` should be aligned with `properties`
* Is the `properties.default.type` section really needed?